lms/models/lti_user.py

```diff
@@ -1,8 +1,26 @@
 from typing import NamedTuple
 
+import sqlalchemy as sa
+
+from lms.db import BASE
 from lms.models import HUser
 
 
+class _LTIUser(BASE):
+    __tablename__ = "lti_user"
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    lms_id = sa.Column(
```

Comment:
I added the FK to lm_is in some place but didn't add the `relationship`, no reason, just lazyness